### DISCUSSION TITLE
How to Send Request With Cookies? credentials = true

### DISCUSSION THREAD

**Discussion by SefaCrenno** [refinedev/refine#6510] [Q&A] [answered]
Hello, I am wondering how to put cookies to request from refine app to some API. I have tried many ways but still not able send request with cookies. I will provide few codes to explain it better:

import { useTable } from '@refinedev/react-table';

const {
....
  } = useTable({
    columns,
    refineCoreProps: {
      metaData: {
        headers: {
          Authorization: 'Bearer ' + identity?.authToken,
        },
        credentials: 'include',
      },
    },
    
    
    
    
   Another different method I use:
    const {
.....
{
  useList,
} from '@refinedev/core';

  } = useList<IComponentOptions, HttpError>({
    resource: 'components',
    meta: {
      headers: {
        Authorization: 'Bearerx ' + identity?.authToken,
      },
      credentials: 'include',
    },
  });

**Comment by BatuhanW** [comment, marked as answer]
Hello @SefaCrenno you can swizzle your data provider and modify your logic for making requests. https://refine.dev/docs/data/packages/simple-rest/#customizing-the-data-provider-

For your specific use-case, we have AI Chatbot in our [documentation](https://refine.dev/docs/) that can help you.
